src/send.jl

```diff
@@ -43,3 +43,27 @@ function sendattachmenttoslack(data, user_endpoint::String)
 
     return (String(r.body))
 end
+
+"""
+    sendfiletoslack(filepath, channel, token)
+
+Sends a file with a message to the specified channel.
+https://api.slack.com/methods/files.upload
+"""
+function sendfiletoslack(filepath, message, channel, token)
+    HTTP.setuseragent!("HTTP.jl")
```

Comment:
I just copied this line from other functions, was this unnecessary?

Reply:
Not 100% sure tbh, I would have to go back and dig around